### ISSUE TITLE
`umoci unpack` doesn't work generated with Skopeo OCI transport

### ISSUE THREAD

**Issue by giuseppe** [openSUSE/umoci#136]
here a simple reproducer.  The error seems to be caused by not having a `refs` directory in the fetched image.

```bash
$ skopeo copy docker://busybox:latest oci:busybox:latest
Getting image source signatures
Copying blob sha256:27144aa8f1b9e066514d7f765909367584e552915d0d4bc2f5b7438ba7d1033a
 624.00 KB / 682.85 KB [==================================================>----]
 682.85 KB / 682.85 KB [=======================================================]Copying config sha256:74c4af6c88707fb5b27aab24c5b3425a090e4a2f299df6e28f6894c8a529516a
 0 B / 575 B [-----------------------------------------------------------------]
Writing manifest to image destination
Storing signatures

$ ls busybox/
blobs  index.json  oci-layout

$ umoci unpack --image busybox:latest extracted
   ⨯ open CAS: validate: check refdir: invalid image detected

```

**Comment by cyphar** [comment]
@giuseppe That's because `umoci` currently doesn't support `1.0.0-rc5` of the image-spec. I have a PR open that should fix this (#133) but there are several higher level design issues I need to work out before we merge it.

**Comment by yuvipanda** [comment]
Can confirm this works in master now.

**Comment by cyphar** [comment]
Yeah, this was fixed with #133 and then subsequently #143 for `1.0.0-rc7` of the `image-spec`.
